src/sagas/projects.js

```diff
@@ -105,11 +106,20 @@ function generateProjectKey() {
   return (date.getTime() * 1000 + date.getMilliseconds()).toString();
 }
 
+export function* projectExported({payload: {exportType}}) {
+  if (exportType === 'repo') {
+    const state = yield select();
+    yield call(saveCurrentProject, state);
+  }
+  yield call(noop);
```

Comment:
Is there a rule that sagas need to yield at least one effect, or somesuch?

Reply:
it appears there is not. but unrelated, I just ran through a test after making this change and it looks like the reponame isn't getting persisted to firebase right, so I'm going to figure out what's going on before sending this back your way with this change